spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/log/SpanLogger.java

```diff
@@ -14,27 +14,37 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.sleuth.event;
+package org.springframework.cloud.sleuth.log;
 
 import org.springframework.cloud.sleuth.Span;
 
 /**
- * <b>sr</b> - Server Receive. The server side got the request and will start processing it.
- * If one subtracts the cs timestamp from this timestamp one will receive the network latency.
+ * Contract for implementations responsible for logging Spans
  *
- * @author Spencer Gibb
- * @since 1.0.0
+ * @author Marcin Grzejszczak
  *
- * @see ClientSentEvent
+ * @since 1.0.0
  */
-@SuppressWarnings("serial")
-public class ServerReceivedEvent extends SpanParentContainingEvent {
+public interface SpanLogger {
```

Comment:
Yes for now MDC is the only impl other than NoOp. Maybe someone wants to add it for commons or sth (I don't know if that's even possible)